File: PHP/Figures exercise/figures/circle.php
<?php
class Circle extends FigureGeometric {

    private float $radio;
    private float $pi;
    private float $calculate;
    
    public function __construct(float $radio){
        parent::_construct("Circle");
        $this->radio = $radio;
        $this->pi = 3.1416;
    }
    public function getRadio ():float{
        return $this-> radio;
    }
    public function  getPi():float{
        return $this-> pi;
    }
    public function calculateArea():float{
        $this->calculate = $this->getRadio()* $this->getRadio()* $this->getPi();
        return $this->calculate;
    }
}

File: JAVA/Figures exercise/com/figures/Circle.java
package com.figures;

public class Circle extends FigureGeometric {
    
    private double radio;
    private double pi;
    private double calculate;
    
    public Circle(double radio){
        super("Circle", "Circulo");
        this.radio = radio;
        this.pi= 3.1416;
     
    }
    
    public double getRadio(){
        return this.radio;
    }
    public double getPi(){
        return this.pi;
    }
    
    public double calculateArea(){
        this.calculate = (this.getRadio() * this.getRadio())* this.getPi();
        return this.calculate;
    }
}